src/home/index.jsx

```diff
@@ -1,27 +1,27 @@
-import React, { Component } from 'react'
-import { Product } from '../components/Product'
+import React from 'react'
+import Layout from '../layouts/Layout'
+import { connect } from 'react-redux'
+import ProductList from '../components/ProductList'
 
-export default class Home extends Component {
-  render() {
-    return (
+function Home({ products }) {
```

Comment:
Let's rename this file to `index.js`. And remove all of the code related to `component` out into `Home.jsx`. The reason behind this is. `index.js` usually for connecting a component to `Redux Store`, it's a best practice.